src/_posts/platform/app/2000-01-01-procfile.md

```diff
@@ -9,48 +9,150 @@ Procfile support is one of our
 [compatibility layers]({% post_url platform/getting-started/2000-01-01-heroku-compatibility %})
 that makes Scalingo compatible with Heroku.
 
-## Definition
+## About the Procfile
 
-The Procfile is a way to define how the platform will try to start your containers.
-In this file, you can define all the different process you would like to run in your
-project **from the same code base**. It is commonly used to define how to start workers which will consume
-asynchronous jobs.
+The **Procfile** is a simple text file specifying how the platform should start
+the Managed Runtime(s) of your application.
 
-A Procfile is a text file using the YAML format describing how to start
-each type of container for your app. It has to be located at the root of your project.
-Environment variables will be interpolated.
+It allows to declare the commands executed in the Managed Runtime(s) on
+startup.
 
-```yaml
-web: bundle exec rails server -p $PORT -e $RAILS_ENV -b 0.0.0.0
-worker: bundle exec sidekiq -e $RAILS_ENV
+Each of these commands is associated to a **process type**, which is basically
+a name that uniquely identifies the type of process.
+
+You can define as many process types as you need for your application, **from
+the same code base**.
+
+Some common use cases for process types:
+- define a process type called `web` to start a process handling HTTP requests.
+- define a process type called `worker` to handle some asynchronous
+  computation.
+- define a `clock` process type to schedule some recurring jobs.
+- define a `console` process type to boot up [one-offs]({% post_url platform/app/2000-01-01-tasks %})
+  and interact with a database from the CLI.
+- ...
+
+When starting an application, Scalingo reads the Procfile and starts at least
+one Managed Runtime for each process type defined (unless it is scaled to 0).
```

Comment:
Except for web, all process are scaled to 0 by default.

Reply:
OK, I made some changes in https://github.com/Scalingo/documentation/pull/2020/commits/64da5f624d74d705bb2f074587d77ad065c7211c to be more explicit about this.

Reply:
> When starting an application, Scalingo reads the Procfile and starts at least
one Managed Runtime for each process type defined (unless it is scaled to 0).

It's still false. By default Scalingo does not start any runtime for new entries in the Procfile.
By default the platform will consider that every process types are scaled to 0:M and the user need to manually scale it up. 

The only exception are web process which by default are scaled to 1:M.

Reply:
Mmh I see.
The idea behind this sentence was to explain the link between the process type and the Managed Runtime. But I agree that it can indeed be misleading, hence the `unless it is scaled to 0` precision.

In https://github.com/Scalingo/documentation/pull/2020/commits/64da5f624d74d705bb2f074587d77ad065c7211c, I'm suggesting to:
- remove the sentence
- replace it with some other explanations, which I found to be better in the end
- add some more precise explanations in the "Starting and Stopping a Process type" section.

Please let me know what you think about it :)

Reply:
Oh indeed i did not understand this sentence in that way.
It sounds good to me.

Nitpick: wouldn't "Creating and removing Process types" be a better title ?

Reply:
> Nitpick: wouldn't "Creating and removing Process types" be a better title ?

You mean "Creating a Process Type" instead of "Defining a Process Type"?